openedx/features/smart_referral/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+from django.views.generic import TemplateView
+
+urlpatterns = [
+    url(r'^oauth-proxy/$',
```

Comment:
I believe adding a comment here to describe the purpose of this proxy url would be helpful to avoid hassle of going to cloudSpounge documentation to see what is it for.